doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,788 @@
+ * Deprecation announcements for unused benchmark convenience scripts:
+   * `pbench-run-benchmark`, `pbench-cyclictest`, `pbench-dbench`, `pbench-iozone`, `pbench-migrate`, and `pbench-netperf`
+ * Many, many, bug fixes and behavioral improvements
+
+You can review the [**Full ChangeLog**](https://github.com/distributed-system-analysis/pbench/compare/b0.69-bp...v0.71.0-alpha.0) on GitHub (all 550+ commits, tags `b0.69-bp` to `v0.71.0-alpha.0`), or read a summary with relevant details below.
+
+We did not bump the "major" release version number with these changes because we still don't consider all the necessary functionality in place for such a major version bump.
+
+Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
+
+
+## The New "Tool Meister" Sub-System
+
+The "Tool Meister" sub-system (introduced by PR #1248) is the major piece of functionality delivered with the release of `v0.71` of the pbench-agent.
+
+This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except one per host to orchestrate the creation of the Tool Meister instance.
+
+One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark script is run.  The Tool Data Sink is responsible for collecting and storing locally all data send to it created by the Tool Meister instances.
+
+### User Orchestration of "Tool Meister" Sub-System
+
+Container images are provided for the constituent components of the Tool Meister sub-system, the Tool Meister image and the Tool Data Sink image.  The images allow for the orchestration of the Tool Meister sub-system to be handled by the user instead of automatically by the pbench-agent.
+
+### The "Tool Meister" Sub-System with No Tools
+
+While this is not a new feature of the Pbench Agent, it is worth noting that when no tools are registered, the "Tool Meister" sub-system is not deployed and the bench scripts still execute normally.
+
+
+## All Tool Registration Handled Locally
+
+Along with the new "Tool Meister" sub-system comes another subtle, but significant, change to how tools are registered.
+
+With the v0.71 release, the record of which tools are registered on which hosts are kept local to host on which `pbench-register-tool` or `pbench-register-tool-set` are invoked.
```

Comment:
"local to *the* host on which"?? Also, this is "the record [...] are kept", and it should really be "is kept". (Or "the records [...] are kept", I suppose.)